app/Http/Controllers/AdminTopController.php

```diff
@@ -3,8 +3,18 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
-
+use App\Models\Contact;
+use Illuminate\Support\Facades\DB;
```

Comment:
```suggestion
use App\Models\Contact;
use Illuminate\Http\Request;
use Illuminate\Support\Facades\DB;
```
use するクラスはアルファベット順にしたほうが可読性あがりますよ